### ISSUE TITLE
Default values are not displayed for types other than string

### ISSUE THREAD

**Issue by radao** [mkdocstrings/python#8]
**Describe the bug**
Using type hints and enabling `show_signature_annotations` displays each parameters type correctly, but not default value unless its a string. 

**To Reproduce**
1. Define a method such as `def foo(a: str = 'hello', b: int = 10, c: bool = False, d: str = 'world') -> None`
2. Enable`show_signature_annotations` in your `mkdocs.yml`:
```
plugins:
  - mkdocstrings:
      handlers:
        python:
          rendering:
            show_signature_annotations: True
```
3. Observe in the generated docs only the string fields default values are displayed (ie `a` and `d`).

**System (please complete the following information):**
- `mkdocs` versions:
```
mkdocs==1.2
mkdocstrings==0.18.1
mkdocstrings-python==0.6.6
```
- Python version: 3.7
- OS: Linux

**Comment by pawamoy** [comment]
Confirmed on Python 3.7 (greater versions seems unaffected).

**Comment by pawamoy** [comment]
Fixed in Griffe 0.14.1
